test/unit/dialects/mssql/query-generator.test.js

```diff
@@ -171,5 +171,34 @@ if (current.dialect.name === 'mssql') {
         mssql: 'ALTER TABLE [myTable] DROP [myColumnKey]'
       });
     });
+
```

Comment:
looks good, beside we have integration tests for increment/decrement so its verified